packages/kolibri-common/components/SyncSchedule/constants.js

```diff
@@ -0,0 +1,8 @@
+export const SyncPageNames = {
+  MANAGE_SYNC_SCHEDULE: 'MANAGE_SYNC_SCHEDULE',
+  EDIT_SYNC_SCHEDULE: 'EDIT_SYNC_SCHEDULE',
+};
+
+// These are copied from (facility | device)/../constants.js' PageNames
+export const FacilityReturnPage = 'DATA_EXPORT_PAGE';
+export const DeviceReturnPage = 'FACILITIES_PAGE';
```

Comment:
It doesn't look like you are using Line 7 and 8 anywhere in your code. It's probably better to get rid of it.